src/main/java/org/openmainframeproject/cobolcheck/services/cobolLogic/Interpreter.java

```diff
@@ -163,6 +163,27 @@ public static boolean isEndOfStatement(CobolLine currentLine, CobolLine nextMean
         if (containsOnlyPeriod(nextMeaningfulLine)) {
             return false;
         }
+        if (currentLine.containsToken(Constants.CALL_TOKEN)) {
+            List<String> currentTokens = currentLine.getTokens();
+            int callTokenCount = 0, endCallTokenCount = 0;
+            for (String token : currentTokens) {
+                if (token.equals(Constants.CALL_TOKEN)) {
+                    callTokenCount++;
+                }
+                if (token.equals(Constants.END_CALL_TOKEN)) {
+                    endCallTokenCount++;
```

Comment:
I think this might break if the "end-call-token" is replaced with a period.

Try making this test: 
CALL "PROGRAM" USING DATA-1 ON EXCEPTION 
DISPLAY "ERROR".
CALL "PROGRAM" USING DATA-1 ON EXCEPTION 
DISPLAY "ERROR".
What code does that result in?
It should look something like this: 
* CALL "PROGRAM" USING DATA-1 ON EXCEPTION DISPLAY "ERROR".
* CALL "PROGRAM" USING DATA-1 ON EXCEPTION DISPLAY "ERROR".
But I think it will concatenate it like this:
* CALL "PROGRAM" USING DATA-1 ON EXCEPTION DISPLAY "ERROR". CALL 
* "PROGRAM" USING DATA-1 ON EXCEPTION DISPLAY "ERROR".

Reply:
@Rune-Christensen 
Thanks for the question.

The `If` conditions above line 166 will take care of the period terminator preventing the code from breaking.

The above code is returning the below code:
```COBOL
*CALL "PROGRAM" USING DATA-1 ON EXCEPTION
*    DISPLAY "ERROR"
           CONTINUE                                                            
           .                                                                 
*CALL "PROGRAM" USING DATA-1 ON EXCEPTION
*    DISPLAY "ERROR".                                       
            CONTINUE                                                            
           .
```
I have also added tests in my new PR :)

Regards,
Akash Kumar